### ISSUE TITLE
Trying to follow your tutorial!

### ISSUE THREAD

**Issue by Omshinwa** [miltoncandelero/pixi-hotwire#6]
Hi, this is probably very basic (i'm very new to this) but after downloading it's telling me that

`Argument of type '{ view: HTMLCanvasElement; resolution: number; autoDensity: boolean; backgroundColor: number; width: number; height: number; }' is not assignable to parameter of type 'IApplicationOptions'.
  Object literal may only specify known properties, and 'view' does not exist in type 'IApplicationOptions'.ts(2345)`

basically the first line of code lol

https://i.imgur.com/MGwnueG.png

What am I doing wrong with typescript...?
